### DISCUSSION TITLE
Glewlwyd in production - what are the risks please?

### DISCUSSION THREAD

**Discussion by grantma** [babelouest/glewlwyd#210] [Q&A]
Hi!

Been looking at what's around, glewlwyd is in Debian, Keycloak is a Java monster.

I am quite happy to run glewlwyd internally to complement Samba AD, but wary about exposing it on the Internet for people to use with their phones and devices.  Do you have any advice please?  

Thank you!

**Comment by babelouest** [comment]
Hi,

I don't know all the risks, that's why I changed its description to experimental. Lately I discovered several issues, and realized this project should not be taken for much than what it is: a personal project, way far from what other SSO endorsed by foundations or companies would provide.

I also have less time and will to work on it, so I would not want people to expect more.

You have several alternatives, some are listed in the [oauth.net website](https://oauth.net/code/), some are less heavy than keycloak, and ready to use in a docker instance, like zitadel.

**Comment by grantma** [comment]
Hi!

Have you thought about getting a developer team organised? The program is
useful as it's already in Debuan and Ubuntu, and ready to go. There's
always quite a bit of pfaffing around to get Java going, which IMHO does
not exist with Glewlwyd.

Thank you!

Matt Grant

Debian Developer
